packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,646 @@
+      "title": "Node bundler",
+      "description": "Function bundling method used in `@netlify/zip-it-and-ship-it`.",
+      "type": "string",
+      "enum": ["zisi", "esbuild", "nft", "none"],
+      "x-taplo": {
+        "docs": {
+          "enumValues": [
+            "Default function bundling method for JavaScript functions.",
+            "Method that leverages `esbuild` to bundle functions, resulting in shorter bundling times and smaller artifacts. TypeScript functions use `esbuild` by default.",
+            "Method that leverages `nft` to bundle functions, resulting in shorter bundling times and smaller artifacts. ESM functions use `nft` by default.",
+            "Disables bundling completely and expects the function to be already bundled and usable."
+          ]
+        },
+        "links": {
+          "key": "https://github.com/netlify/zip-it-and-ship-it"
+        }
+      }
+    },
+    "FunctionConfigExternalNodeModules": {
+      "title": "External node modules",
+      "description": "List of Node.js modules that are copied to the bundled artifact without adjusting their source or references during the bundling process; only applies when `node_bundler` is set to `esbuild`. This property helps handle dependencies that can't be inlined, such as modules with native add-ons.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "FunctionConfigIncludedFiles": {
+      "title": "Included files",
+      "description": "list of additional paths to include in the function bundle. Although our build system includes statically referenced files (like `require(\"./some-file.js\")`) by default, `included_files` lets you specify additional files or directories and reference them dynamically in function code. You can use `*` to match any character or prefix an entry with `!` to exclude files. Paths are relative to the base directory.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    }
+  },
+  "additionalProperties": false,
+  "properties": {
+    "build": {
+      "$ref": "#/definitions/BuildOptions"
+    },
+    "plugins": {
+      "$ref": "#/definitions/Plugins"
+    },
+    "context": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/SpecificBranchContext"
+      },
+      "properties": {
+        "production": {
+          "$ref": "#/definitions/ProductionContext"
+        },
+        "deploy-preview": {
+          "$ref": "#/definitions/DeployPreviewContext"
+        },
+        "branch-deploy": {
+          "$ref": "#/definitions/BranchDeployContext"
+        }
+      }
+    },
+    "redirects": {
+      "type": "array",
+      "default": [{ "from": "/old-path", "to": "/new-path" }],
+      "items": {
+        "title": "Redirects",
+        "description": "You can manage your redirects directly in your `netlify.toml`. For each redirect you want to declare, add an entry with the [[redirects]] heading.",
```

I noticed a comment in the docs about these being global - let's mention that

```suggestion
        "description": "Declare global redirects for all builds. For each redirect you want to declare, add an entry with the [[redirects]] heading. If you need context-specific redirect rules, use a `_headers` or `_redirects` file instead.",
```